src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3DirectivesFactory.java

```diff
@@ -42,37 +37,37 @@ private String buildTaxonomyUrl(Optional<OrganismQualifier> qualifier) {
                 .orElseGet(getOrganism);
     }
 
-    public GFF3Directives.GFF3Species extractSpecies(Entry entry) throws NoSourcePresentException {
+    public GFF3Species extractSpecies(Entry entry) throws NoSourcePresentException {
```

Comment:
Suggestion to rename to:

`public GFF3Species createSpecies(Entry entry) throws NoSourcePresentException {`